npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/vulnerability_auditor.rb

```diff
@@ -87,8 +92,20 @@ def audit(dependency:, security_advisories:)
 
         attr_reader :dependency_files, :credentials
 
-        def viable_audit_result?(audit_result, security_advisories)
-          validation_result = validate_audit_result(audit_result, security_advisories)
+        def explain_fix_unavailable(validation_result, fix_unavailable)
+          fix_unavailable["explanation"] =
+            case validation_result
+            when :fix_unavailable, :dependency_still_vulnerable, :downgrades_dependencies
+              "No patched version available for #{fix_unavailable['dependency_name']}"
+            when :vulnerable_dependency_removed
+              "#{fix_unavailable['dependency_name']} was removed in the update. Dependabot is unable to " \
+                "open a pull request. Please upgrade manually"
```

Comment:
If I understood correctly, this case is not currently supported, but we do plan to eventually support it? If that's the case, should we make it more clear to the user? I'm thinking something along the lines of 

> Dependabot is not currently able to deal with this situation, but you can still upgrade manually.

Reply:
That's correct and eventually we will remove this case.